sdk/core/Azure.Core/tests/public/Internal/ModelSerializerHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.ClientModel;
+using System.Net.ClientModel.Core;
+using System.Runtime.CompilerServices;
+
+namespace Azure.Core.Tests.Public
+{
+    internal static class ModelSerializerHelper
```

Comment:
You're aware this is duplicated?

Reply:
Yes it is duplicate because we don't want this on the public API surface but keeps the churn in the test classes to a minimum if we add it as internal to the test projects.